### DISCUSSION TITLE
[PSD7] Exercise 2.18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#159] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210018723-b968ae66-4803-44ae-8401-dfe9c90729b4.png)

**Comment by Elia-Belli** [comment]
- (a) $Y = \bar B + C$
- (b) $Y = B + \bar C D + AD$
- (c) $Y = \bar A B \bar C D + ABCD + B \bar D E$

![circuit](https://user-images.githubusercontent.com/107944476/211633352-334424fb-a517-4eff-ad65-2d6280ed3bbf.jpg)

**Comment by fraxb04** [comment]
corretto

**Comment by niccolo-fato** [comment]
![photo_2024-11-21_22-03-43](https://github.com/user-attachments/assets/4ffd5bdd-bbc1-4b76-bc8e-f6897048000e)
